<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="62930" started-at="2019-11-15T15:51:12Z" finished-at="2019-11-15T15:52:15Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="62930" started-at="2019-11-15T15:51:12Z" finished-at="2019-11-15T15:52:15Z">
      <class name="TestAutomation.TechnicalTest.UserLogin_With_InValid">
        <test-method status="PASS" signature="Lauch_Url()[pri:0, instance:TestAutomation.TechnicalTest.UserLogin_With_InValid@d8355a8]" name="Lauch_Url" is-config="true" duration-ms="10170" started-at="2019-11-15T15:51:12Z" finished-at="2019-11-15T15:51:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Lauch_Url -->
        <test-method status="FAIL" signature="UserLogin_With_InValid_UserID()[pri:0, instance:TestAutomation.TechnicalTest.UserLogin_With_InValid@d8355a8]" name="UserLogin_With_InValid_UserID" duration-ms="2311" started-at="2019-11-15T15:51:23Z" finished-at="2019-11-15T15:51:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [My account - My Store] but found [Authentication failed]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [My account - My Store] but found [Authentication failed]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at TestAutomation.TechnicalTest.UserLogin_With_InValid.UserLogin_With_InValid_UserID(UserLogin_With_InValid.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UserLogin_With_InValid_UserID -->
      </class> <!-- TestAutomation.TechnicalTest.UserLogin_With_InValid -->
      <class name="TestAutomation.TechnicalTest.UserLogin_WithValid_ID_AddToCart">
        <test-method status="PASS" signature="Lauch_Url_ForLogin_AddToCart()[pri:0, instance:TestAutomation.TechnicalTest.UserLogin_WithValid_ID_AddToCart@28d25987]" name="Lauch_Url_ForLogin_AddToCart" is-config="true" duration-ms="8721" started-at="2019-11-15T15:51:55Z" finished-at="2019-11-15T15:52:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Lauch_Url_ForLogin_AddToCart -->
        <test-method status="PASS" signature="UserLogin_With_Valid_UserID()[pri:0, instance:TestAutomation.TechnicalTest.UserLogin_WithValid_ID_AddToCart@28d25987]" name="UserLogin_With_Valid_UserID" duration-ms="11648" started-at="2019-11-15T15:52:04Z" finished-at="2019-11-15T15:52:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UserLogin_With_Valid_UserID -->
      </class> <!-- TestAutomation.TechnicalTest.UserLogin_WithValid_ID_AddToCart -->
      <class name="TestAutomation.TechnicalTest.NewUser_Registration">
        <test-method status="PASS" signature="Lauch_Url_for_Registration()[pri:0, instance:TestAutomation.TechnicalTest.NewUser_Registration@59fa1d9b]" name="Lauch_Url_for_Registration" is-config="true" duration-ms="8304" started-at="2019-11-15T15:51:25Z" finished-at="2019-11-15T15:51:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Lauch_Url_for_Registration -->
        <test-method status="FAIL" signature="New_Registration()[pri:0, instance:TestAutomation.TechnicalTest.NewUser_Registration@59fa1d9b]" name="New_Registration" duration-ms="21716" started-at="2019-11-15T15:51:33Z" finished-at="2019-11-15T15:51:55Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id='id_gender1']"}
  (Session info: chrome=78.0.3904.97)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IB', ip: '192.168.87.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_201'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.97, chrome: {chromedriverVersion: 78.0.3904.70 (edb9c9f3de024..., userDataDir: C:\Users\Ibrahim\AppData\Lo...}, goog:chromeOptions: {debuggerAddress: localhost:57585}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 1b8f2f1b2baddefb71d983c7a09dde9d
*** Element info: {Using=xpath, value=//*[@id='id_gender1']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id='id_gender1']"}
  (Session info: chrome=78.0.3904.97)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IB', ip: '192.168.87.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_201'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.97, chrome: {chromedriverVersion: 78.0.3904.70 (edb9c9f3de024..., userDataDir: C:\Users\Ibrahim\AppData\Lo...}, goog:chromeOptions: {debuggerAddress: localhost:57585}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 1b8f2f1b2baddefb71d983c7a09dde9d
*** Element info: {Using=xpath, value=//*[@id='id_gender1']}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
	at TestAutomation.TechnicalTest.NewUser_Registration.New_Registration(NewUser_Registration.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- New_Registration -->
      </class> <!-- TestAutomation.TechnicalTest.NewUser_Registration -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
